-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 2, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 5.377 sec <<< FAILURE!
test1(com.solvd.advancedautomation.base.login.LoginTest)  Time elapsed: 0.005 sec  <<< FAILURE!
java.lang.AssertionError: Failing test1 on purpose
	at org.testng.Assert.fail(Assert.java:111)
	at com.solvd.advancedautomation.base.login.LoginTest.test1(LoginTest.java:10)

test2(com.solvd.advancedautomation.base.login.LoginTest)  Time elapsed: 0.001 sec  <<< FAILURE!
java.lang.AssertionError: Failing test2 on purpose
	at org.testng.Assert.fail(Assert.java:111)
	at com.solvd.advancedautomation.base.login.LoginTest.test2(LoginTest.java:15)

